include/rocksdb/rate_limiter.h

```diff
@@ -40,10 +40,10 @@ class RateLimiter {
   // REQUIRED: bytes_per_second > 0
   virtual void SetBytesPerSecond(int64_t bytes_per_second) = 0;
 
-  // This API allows user to dynamically change the max bytes can be granted in
-  // a single refill period (i.e, burst)
+  // This API allows user to dynamically change the max bytes can be granted at
+  // once. Zero is a special value meaning the number of bytes per refill.
```

Comment:
So "at once" essentially means "per request" now, instead of "at one refill" it used to be. Maybe note the idea of "per request" more explicitly in the comment here?

Reply:
Good point, done.